docs/sovity-edc-api-wrapper.yaml

```diff
@@ -507,6 +507,47 @@ components:
         transportMode:
           type: string
           description: Transport Mode
+        sovereign:
+          type: string
+          description: The publisher and sovereign can be distinct
```

Comment:
"The sovereign is distinct from the publisher by being the legal owner of the data."